Adding options to install idrac-packages and "all" srvadmin-packages.

palsveningson opened the pull request:
The idrac-packages is nice to have installed for racadm-usage.
The "all"-option is a nice feature.

Commit:
Addins options to install idrac-packages and "all" srvadmin-packages.

The idrac-packages is nice to have installed for racadm-usage.
The "all"-option is a nice feature.

vide reviewed (changes requested):
First of all, thanks for your contribution!
Can you remove any RPM mention from the README, since the module targets Debian too?

Thanks!

Commit:
Replacing mentions of RPM with packages.

palsveningson commented:
Hi! Good point, I replaced it with packages.

BR!

vide commented on manifests/install.pp:
```diff
@@ -61,4 +67,19 @@
     }
   }
+  
+  if ( str2bool("${::omsa::install_idac}")) {
```
There's a typo here

vide commented on README.md:
```diff
@@ -99,6 +99,15 @@ As a last note, you should know that if you want to customize the SNMP installat
  Force OMSA installation even when $manufacturer is not Dell. Default: false
 
+ * `install_idrac`
+ Install the meta package-idrac package. Default: false
```
I think `Install the idrac meta package` is better

vide reviewed (changes requested):
Ok, thinking it twice (a part from the other small fixes I commented) I think you should make the `srvadmin-all` package installation mutually exclusive with the other packages, with the current code, dependencies will break if by mistake you have `install_all` and some other install flag activated

Commit:
Updated module with requested changes

Updated some installation-options to be conditional and did requested changes in readme.

palsveningson commented:
Hi! I did some changes. Hope this is what you wanted :)

BR

vide commented:
Hello @palsveningson sorry for the delay. In the end I've implemented your proposal with #3 so I'm gonna close this PR and merge the other one. Please checks if it does what you needed!
